services/client/src/components/video/icon.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const getSourceIcon = source => {
+  if (source === "youtube")
+    return "https://upload.wikimedia.org/wikipedia/commons/thumb/7/72/YouTube_social_white_square_%282017%29.svg/2000px-YouTube_social_white_square_%282017%29.svg.png";
+  return undefined;
+};
+
+const sharedStyles = `
+  text-align: center;
+  margin-top: 0.5rem;
+  width: 35px;
+`;
+
+const Icon = ({ source }) => {
+  const sourceImage = getSourceIcon(source);
+  if (sourceImage) {
+    return (
+      <React.Fragment>
+        <style jsx>{`
+          img {
+            ${sharedStyles};
+          }
+        `}</style>
+        <img src={sourceImage} alt={`${source} icon`} />
+      </React.Fragment>
+    );
+  }
+
+  return (
+    <React.Fragment>
+      <style jsx>{`
+        i {
+          ${sharedStyles};
+        }
+      `}</style>
+      <i className="fa fa-calendar-o" alt="Video icon" />
```

Comment:
There's a `FontAwesome` component available as an existing dependency
https://www.npmjs.com/package/react-fontawesome

Reply:
Punting on this for now until we have a need for more icons; If we proceed with a sprite sheet for the few icons we require, we can probably drop font-awesome.